srv/cmd/main.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main implements a simple HTTP/JSON REST example.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/abcxyz/abc-updater/srv/pkg"
+
+	"github.com/abcxyz/pkg/logging"
+	"github.com/abcxyz/pkg/renderer"
+	"github.com/abcxyz/pkg/serving"
+)
+
+const defaultPort = "8080"
+
+var port = flag.String("port", defaultPort, "Specifies server port to listen on.")
```

Comment:
This is fine for a simple server like this. But in general we prefer envvar for configurations.

Reply:
I actually have envvar, forgot to migrate this from the the `abc` http server template

Reply:
Envconfig is added in #63 , resolving here and will move port to envconfig there